pkg/v21/key/error_internal.go

```diff
@@ -0,0 +1,21 @@
+package key
```

Comment:
Damn this is due to the controller per provider design again. But I don't think this should be in the key package.

Could you move it to `error_internal.go` in `github.com/giantswarm/cluster-operator/pkg/v21/chartconfig`? Ideally we wouldn't export the error matcher but we have to in this case.